contracts/adapters/axelar/AxelarReceiverAdapter.sol

```diff
@@ -0,0 +1,116 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "../../interfaces/IBridgeReceiverAdapter.sol";
+import "./interfaces/IAxelarChainRegistry.sol";
+import "./interfaces/IAxelarGateway.sol";
+import "./interfaces/IAxelarExecutable.sol";
+import "./libs/StringAddressConversion.sol";
+
+contract AxelarReceiverAdapter is
+    Ownable,
+    IAxelarExecutable,
+    IBridgeReceiverAdapter
+{
+    mapping(uint256 => address) public senderAdapters;
```

Comment:
For our purpose (i.e. for MMA V1 that is optimised for Uniswap's use case) we can maintain a single sender since the sender will always be Ethereum.